modules/nf-core/polypolish/polish/main.nf

```diff
@@ -0,0 +1,50 @@
+process POLYPOLISH {
+    tag "$meta.id"
+    label 'process_high_memory'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/polypolish:0.6.0--hdbdd923_0':
+        'biocontainers/polypolish:0.6.0' }"
+
+    input:
+    // input:  polypolish polish [OPTIONS] <ASSEMBLY> [SAM]
+    tuple val(meta), path(fasta)
+    tuple val(meta), path(sam)
+
+    output:
+    // polypolish polish draft.fasta filtered_1.sam filtered_2.sam > polished.fasta
+    tuple val(meta), path("*.fasta"), emit: fasta
```

Comment:
You may need to add an additional output channel (optional):

```
      --debug <DEBUG>
          Optional file to store per-base information for debugging purposes
```

In this case as there isn't a default file name/suffix, you probably want to control it's creation name your self, but adding an extra input channel e.g. `val save_debug`

Then output `tuple val (meta), path(*.txt), optional: true, emit: debug`

then in `script:` having `def debug_mode = save_debug ? '--debug ${prefix}.txt' : ''